test/sub.mk

```diff
@@ -1,6 +1,6 @@
 # The default target is `test`, which runs in 3 phases
 # 1. Remove the compile.stamp file if the compiler jar has changes
-# 2. If COMPILER_OUTPUT is ll: compile the .bal test cases into .ll files 
+# 2. If COMPILER_OUTPUT_EXTENSION is .ll: compile the .bal test cases into .ll files 
```

Comment:
How does this get set? Does the user have to say `make COMPILER_OUTPUT_EXTENSION=.o" in the JNI case?

Reply:
Yes
For running tests with JNI (jar file, since it is the default), it is `make test COMPILER_OUTPUT_EXTENSION=.o` 
If you need to use the executable (one from Graal VM) file, it also needs to be specified as `make test COMPILER_OUTPUT_EXTENSION=.o O_COMPILER=../../../testbuild/target/bin/nballerina`

Reply:
Changed above two commands to `make test LINK_FILE_EXTENSION=.o` and `make test LINK_FILE_EXTENSION=.o O_COMPILER=../../../testbuild/target/bin/nballerina` respectively